apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/AssertUtils.java

```diff
@@ -0,0 +1,52 @@
+package gov.cms.bfd.server.war.utils;
+
+import static org.junit.Assert.fail;
+
+import java.util.Objects;
+
+public class AssertUtils {
+
+  @FunctionalInterface
+  public interface Executor {
+    void execute();
+  }
+
+  public static Exception catchExceptions(Executor executor) {
+    Exception ex = null;
+
+    try {
+      executor.execute();
+    } catch (Exception e) {
+      ex = e;
+    }
+
+    return ex;
+  }
+
+  public static void assertThrowEquals(Exception expected, Exception actual) {
+    if (expected.getClass() == actual.getClass()) {
```

Comment:
A check for `actual == null` here could print `fail` with a message indicated that a required exception was not thrown.  As it is this would throw an NPE if the expected exception was not thrown.

Reply:
Good catch, I did forget the nullcheck